scripts/legacy/make_beatport_key.py

```diff
@@ -4,7 +4,7 @@
 import os
 
 
-beatport_key_INDEX_PATH = '../mirdata/indexes/beatport_key_index.json'
+beatport_key_INDEX_PATH = '../../mirdata/datasets/indexes/beatport_key_index.json'
```

Comment:
can you move this script out of scripts/legacy and into just scripts since you updated it to the new api? :)

Reply:
Sure! I forgot to change it afterward. Thanks for the note!!